user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/dedicated_clustered_storage/Configuring_database/Configuring_multiple_Elasticsearch_clusters.md

```diff
@@ -125,6 +125,17 @@ To configure this setup:
 1. After the file has been added, restart DataMiner. *DBConfiguration.xml* will now take precedence over the Elasticsearch configuration defined in the file *DB.xml*.
 
 > [!NOTE]
-> If an exception occurs for one of the replicated clusters, an alarm will be generated in the Alarm Console, indicating that not all data might be replicated. If further errors occur, no new alarms are created until the DMA is restarted.
+> If the replicated cluster is still up and running, but an exception occurs for one of the replicated clusters, an alarm will be generated in the Alarm Console, indicating that not all data might be replicated. If further errors occur, no new alarms are created until the DMA is restarted.
 
 <!--***-->
+
+## Troubleshooting
+
+During normal operation, DataMiner will offload the data written to both Elasticsearch clusters simultaneously.
+
+If one of the clusters goes down, an error will be displayed in the Alarm Console to indicate that the indexing cluster is down: `All nodes in the indexing cluster are down` and/or `All nodes in the replicated indexing cluster [IP1, IP2,...] are down`.
+
+If this happens, you need to alter the *priorityOrder* (see above) so that the backup indexing cluster becomes active. Specifically, this means setting *priorityOrder="0"* where the *priorityOrder* used to be 1 and the other way around. After you have done so, restart DataMiner. This should restore DataMiner to a functional state.
+
+> [!IMPORTANT]
+> If the main Elasticsearch cluster goes down, it will need to be restored afterwards. To ensure that the data is not in sync, first follow the procedure on [Verifying Elasticsearch synchronization](xref:Verifying_Elasticsearch_Synchronization#checking-database-health), then follow the procedure to restore on [Taking a snapshot of one Elasticsearch cluster and restoring it to another](xref:Taking_snapshot_Elasticsearch_cluster_and_restoring_to_different_cluster).
```

Comment:
"To ensure that the data is **not** in sync" => Is this correct? It seems a bit odd that you would ensure that the data is not synced.

Reply:
That is an oversight from my part. The data needs to be in sync indeed.